api/api/settings.py

```diff
@@ -32,12 +32,12 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-
```

Comment:
Keep the blank line here, to create a separation between django apps and custom/3rd party apps.